components/brave_wallet/browser/internal/hd_key_ed25519_unittest.cc

```diff
@@ -18,7 +34,6 @@ TEST(HDKeyEd25519UnitTest, TestVector1) {
 
   // m
   auto master_key_base = HDKeyEd25519::GenerateFromSeed(bytes);
-  EXPECT_EQ(master_key_base->GetPath(), "m");
```

Comment:
We don't need `path` property at all

Reply:
Why don't we need this anymore? I believe there's been cases where different paths are used in ledger and phantom for example where the user needs to be able to choose their path in order to make sure they can recover their keys. If we can't choose the path won't that remove that capability?

Reply:
This one being removed(and sec256k1 similar one to be removed) are referenced by tests only. 
HW wallets use hd paths for communications with device and we don't derive anything with them. 
These are two different things with same name. The second one is not touched